src/aead/aes.rs

```diff
@@ -192,15 +57,15 @@ impl Key {
             // SAFETY: `vpaes_set_encrypt_key` satisfies the `set_encrypt_key!`
             // contract for these target architectures.
             Implementation::VPAES_BSAES => unsafe {
-                set_encrypt_key!(vpaes_set_encrypt_key, bytes, &mut key, cpu_features)?;
+                set_encrypt_key!(vpaes_set_encrypt_key, bytes, cpu_features)
             },
 
             // SAFETY: `aes_nohw_set_encrypt_key` satisfies the `set_encrypt_key!`
             // contract.
             Implementation::NOHW => unsafe {
-                set_encrypt_key!(aes_nohw_set_encrypt_key, bytes, &mut key, cpu_features)?;
+                set_encrypt_key!(aes_nohw_set_encrypt_key, bytes, cpu_features)
             },
-        };
+        }?;
```

Comment:
This isn't reachable because the branch where the called function returns an error isn't reachable. No different tha before. We'll deal with this in a future change.